src/frontend/src/lib/components/dapps/DappPromoBanner.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+	import Button from '$lib/components/ui/Button.svelte';
+	import Img from '$lib/components/ui/Img.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import type { DApp } from '$lib/types/dapp';
+
+	export let dApp: DApp;
+	export let backgroundImage: string;
+</script>
+
+<article class="relative flex h-64 items-end overflow-hidden rounded-2xl">
+	<div class="absolute">
+		<Img src={backgroundImage} alt={dApp.name} />
+	</div>
+	<div class="flex-1 px-4 py-4 backdrop-blur-sm">
+		<div class="flex items-center gap-x-2">
+			<div class="h-12 w-12 rounded-full">
+				<Img src={dApp.logo} alt={dApp.name} />
```

Comment:
You might want to revisit the `alt` here. The app name isn't really an accessible text I would say. It should probably be a text that contains the app name.